packages/material-ui/src/Portal/index.d.ts

```diff
@@ -1,2 +1,5 @@
-export { default } from './Portal';
-export * from './Portal';
+import { Portal } from '@material-ui/unstyled';
+
+export { PortalProps } from '@material-ui/unstyled';
+
+export default Portal;
```

Comment:
I think that we could blindly reexport everything.

Reply:
Same error above with export